### ISSUE TITLE
server not responding anymore - Preview function down

### ISSUE THREAD

**Issue by rarcif** [Dhaiwat10/react-link-preview#26]
both customFetcher as well as original library can not connect to server. Preview is currently not working e.g. check demo page https://codesandbox.io/s/rlp-demo-90e1x?file=/src/App.js

**Comment by Dhaiwat10** [comment]
It's working for me though:
![Uploading CleanShot 2022-05-26 at 17.40.49.png…]()

**Comment by rarcif** [comment]
yes now it does on my end as well. Any idea what could have caused this? From the console log it seemed there was a get connection error.

**Comment by Dhaiwat10** [comment]
probably too many requests on the public server.
